mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
but will that work if the user sets something else that isn't one of mayas string arguments.
exampel 
maya accepts "12fps" as an option
but it wont accept "25fps" as an option
to get 25 fps you need to enter "pal"